endoreg_db/views/examination_views.py

```
@@ -9,38 +9,85 @@
 
-        choices = FindingMorphologyClassificationChoice.objects.filter(
-            classification__in=FindingMorphologyClassification.objects.filter(
-                classification_type__in=list(all_classification_types)
-            )
-        ).distinct()
+@api_view(["GET"])
+def get_location_choices_for_classification(request, exam_id, location_classification_id):
+    """
+    Retrieves location choices for a specific location classification within an examination.
+    """
+    exam = get_object_or_404(Examination, id=exam_id)
```

Ruff (0.11.9)

Local variable `exam` is assigned to but never used

Remove assignment to unused variable `exam`

(F841)